application/controllers/admin/limereplacementfields.php

```diff
@@ -446,16 +446,16 @@ private function collectQuestionReplacements($surveyid, $gid = null, $qid = null
         if ($gid != null && $qid == null) {
             $oGroup = QuestionGroup::model()->findByPk($gid);
             $oCriteria->with = ['group'];
-            $oCriteria->compare('group_order', $oGroup->group_order, false, '<=');
+            $oCriteria->compare('group_order', '<='.$oGroup->group_order,);
```

Comment:
Cast to int?

Reply:
Don't think : 
See Yii code

$value is set here https://github.com/yiisoft/yii/blob/d01658e47d799e5778608982a45bfcf2d4062a17/framework/db/schema/CDbCriteria.php#L427

And use PDO param for value : https://github.com/yiisoft/yii/blob/d01658e47d799e5778608982a45bfcf2d4062a17/framework/db/schema/CDbCriteria.php#L447

The original issue seems top be about column name , not value

(I check with my postgre version an no issue. )

Reply:
Yes confirm:

$this->addCondition($column.$op.self::PARAM_PREFIX.self::$paramCount,$operator);

Become 
`$this->addCondition('group_order < :ycp0,'AND');`

And before the previous fix : 
`$this->addCondition('group.group_order < :ycp0,'AND');`

Reply:
`addCondition` is not same as `compare`?

Reply:
Oh, they actually split the value. Good lord.

Reply:
Better  to use array instead of string, then. More clear.

Reply:
> Better to use array instead of string, then. More clear.

You want to rewrite YII ?

In place of compare we can use addCondition directly :) .

Else we can use [Yii::app()->db->quoteColumnName](https://github.com/LimeSurvey/LimeSurvey/blob/eba81f2ab24d2b2484d7b2b1d730df191b573aed/application/models/Quota.php#L181) in addCondition or compare …

Reply:
> You want to rewrite YII ?

No, $value can be both array and string.